tools/FakeItEasy.Build/.gitignore

```diff
@@ -0,0 +1 @@
+ToolPaths.cs

```

Comment:
we don't just add the appropriate line to the top level .gitignore?

Reply:
We could. I thought it would be cleaner this way, but I can change it.

Reply:
I know every time I go back to svn (where every ignore is in the folder) I'm annoyed at having to repeat myself, so perhaps I like the top-level .gitignore too much. We do already ignore some files in lower level directories from that file, so I thought that was our intention going forward. I think I do prefer one ignore file, but if you do want to have a second, shouldn't we have one in src as well?

Reply:
> I think I do prefer one ignore file, but if you do want to have a second, shouldn't we have one in src as well?

And one int tests, for the `.received.txt` files... OK, you got me there. I don't want to go down that road at this point. I'll put everything in the same .gitignore.